src/screens/Documents/View/index.js

```diff
@@ -112,42 +130,182 @@ class DocumentsViewScreen extends NavigatorComponent<Props & DocumentsState & Ac
+        'Document Hash', `${result.value}`,
+        [
+          {
+            text: 'SHARE',
+            //  onPress: () => this.props.navigator.dismissModal(),
+          },
+          {
+            text: 'COPY',
+            onPress: () => this.writeToClipboard(result.value),
+          },
+        ],
+      );
+    } else {
+      console.log('Here');
+    }
+  };
+
+  writeToClipboard = async (value) => {
+    this.setState({ text: value });
+    await Clipboard.setString(this.state.text);
+  };
+
+  txModalClose() {
+    this.setState({ visibleTxModal: false });
+  }
+
   render() {
     const document = getOpenedDocument(this.props);
-    if (document == null) return (<View />);
+    if (document == null) return <View />;
+    const { isUploading } = this.props;
+    const documentDetail = [
+      {
+        name: i18n.t('screens.documentView.registered'),
+        value: document.registered,
+      },
+      {
+        name: i18n.t('screens.documentView.txhash'),
+        value: document.tx_hash,
+      },
+      {
+        name: i18n.t('screens.documentView.signature'),
+        value: document.signature,
+      },
+      {
+        name: i18n.t('screens.documentView.dochash'),
+        value: document.doc_hash,
+      },
+    ];
 
     return (
       <View style={styles.screenContainer}>
         <View style={styles.previewContainer}>
-          <Image source={imageSource(this.content, document.mimeType)} style={styles.preview} resizeMode='contain' />
+          <Image
+            source={imageSource(this.content, document.mimeType)}
+            style={styles.preview}
+            resizeMode='contain'
+          />
         </View>
         <View style={styles.metadataContainer}>
-          <Text style={styles.headline}>
-            {document.name}
-          </Text>
-          <Text style={styles.footnote}>
-            {document.description}
-          </Text>
+          <ScrollView>
+            <Text style={styles.headline}>{document.name}</Text>
+            <Text style={styles.footnote}>{document.description}</Text>
+            {!isUploading && (
+              <Button
+                enabled
+                style={styles.actionButton}
+                title={i18n
+                  .t('screens.documentView.submitdocument')
+                  .toUpperCase()}
+                onPress={this.onDocumentSubmit}
+                styleTitle={styles.settingsText}
+              />
+            )}
+            {isUploading && document.status && (
+              <DocumentListItem
+                id={1}
+                name={i18n.t('screens.documentView.status')}
+                value={document.status}
+                disclosureIconVisible={false}
+                onPress={() => console.log('hello')}
+              />
+            )}
+            {document.tx_hash &&
+              document.doc_hash &&
+              document.signature &&
+              documentDetail.map((index, id) => (
+                <DocumentListItem
```

Comment:
It seems that we have a misunderstanding regarding that component. It's designed to display documents on the list of documents, not to display document details. Please consider creating another component for what you need.

Reply:
@seland Sure, I created a new component now.